test/util/runtime_types_test.cpp

```diff
@@ -0,0 +1,30 @@
+#include "execution/sql/runtime_types.h"
```

Comment:
1. Move to execution sql folder.
2. Can you bring in the tests from the TPL repo, too?
3. Add tests for the corner cases I mentioned earlier.
4. Add tests to exercise the branches in `Timestamp::AdjustTimezone()`.

Reply:
Updated.